src/components/ReportStatus.jsx

```diff
@@ -68,14 +68,14 @@ const ReportStatus = (props, routeProps) => {
 	}
 
 	const showMoreComments = (clickEvent) =>{
-		if (clickEvent.currentTarget.innerText === 'Show more...')
+		if (buttonName === 'Show more...')
 		{
 			setcommentLength(comments.length);
-			clickEvent.currentTarget.innerText = 'Show less...';
+			setButtonName('Show less...');
 		}
 		else{
 			setcommentLength(3);
-			clickEvent.currentTarget.innerText = 'Show more...';
+			setButtonName('Show more...');
```

Comment:
not blocking, but another case where a static string is repeated, prefer to use variables in this situation.